config/environments/production.rb

```diff
@@ -118,6 +118,12 @@
   # config.active_record.database_resolver = ActiveRecord::Middleware::DatabaseSelector::Resolver
   # config.active_record.database_resolver_context = ActiveRecord::Middleware::DatabaseSelector::Resolver::Session
 
+  # HSES API
+  config.x.hses.api = ENV["HSES_API"]
```

Comment:
[suggestion] I don't think this env var name is quite specific enough. What about `HSES_API_HOSTNAME`?